src/content/drafts/README.md

```diff
@@ -0,0 +1,58 @@
+# Write for the ESLint blog
+
+Have you had an interesting experience with ESLint? Want to share it with the rest of the world? Writing a blog post for the ESLint blog helps get the word out and also contributes valuable information to the ESLint community as a whole. 
+
+## How it works
+
+Writing for the ESLint blog works like this:
+
+1. You write an awesome post related to ESLint (ideas below).
+1. The ESLint team helps you edit and revise the post to make it as good as it can be.
+1. You get paid $300 USD for your post.
```

Comment:
The standard hourly rate for team members also applies to the blog post, right?

Reply:
We need to discuss in TSC because we didn’t make this distinction when we decided on this. I’d like it to be $300 regardless.